labs/32_update_hosts.md

```diff
@@ -7,67 +7,66 @@ The config playbook we use to install and configure OpenShift removes yum exclud
 
 First, let's check if the excludes have been set on all nodes. Connect to the first master and run:
 ```
-[ec2-user@master0 ~]$ ansible all -m shell -a "grep exclude /etc/yum.conf"
+[ec2-user@master0 ~]$ ansible nodes -a "grep exclude /etc/yum.conf"
```

Comment:
I'd suggest using the commands `atomic-openshift-excluder status` and `atomic-openshift-docker-excluder status` instead of `grep`. If the output is:
```
# atomic-openshift-excluder status
exclude -- All packages excluded
```

we know everything's fine.

Reply:
Fixed by d28e227258376bbfce15f17f9533123f830f1e0c